ci/scripts/clang-format.sh

```diff
@@ -22,18 +22,6 @@ merge_base="$(git merge-base origin/$tgt_branch HEAD)" || {
 }
 echo "Running C/C++ lint checks on files changed since $merge_base"
 
-TMPFILE="$(mktemp)" || {
-    echo >&2 "Failed to create temporary file"
-    exit 1
-}
-trap 'rm -f "$TMPFILE"' EXIT
+trap 'echo "code failed `clang_format_check` fix with `./bazelisk run //:clang_format_fix`"' ERR
 
-set -o pipefail
-git diff -U0 "$merge_base" -- "*.cpp" "*.cc" "*.c" "*.h" ':!*/vendor/*' | \
```

Comment:
This functionality can be modified to generate a list of diffed files and pass it into clang_format check. This should bring it back to quicklint